test/extended/olm/olmv1.go

```diff
@@ -99,6 +101,43 @@ var _ = g.Describe("[sig-olmv1][OCPFeatureGate:NewOLM][Skipped:Disconnected] OLM
 	})
 })
 
+var _ = g.Describe("[sig-olmv1][OCPFeatureGate:NewOLM][Skipped:Disconnected] OLMv1 Catalog's /v1/api/all endpoint", func() {
+	defer g.GinkgoRecover()
+	oc := exutil.NewCLIWithoutNamespace("default")
+
+	g.It("should serve FBC", func(ctx g.SpecContext) {
+		checkFeatureCapability(ctx, oc)
+
+		// Start port-forward in background
+		_, _, _, err := oc.AsAdmin().Run("port-forward").Args("-n", "openshift-catalogd", "svc/catalogd-service", "8080:443").Background()
+		o.Expect(err).NotTo(o.HaveOccurred())
```

Comment:
I could imagine port-forward hiding issues that in-cluster clients would encounter (e.g. in-cluster DNS).

Doing a port-forward based test is probably a good thing to do if that is functionality we expect to work, but I also think we should run a `Job` workload that can perform a similar test using cluster networking.

Reply:
Yup a `Job` is actually perfect for this use case, and that totally slipped my mind 😮‍💨 updated the PR to use a `Job` instead of port forwarding